pdc/apps/changeset/migrations/0004_changeset_requested_on.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import datetime
+from django.utils.timezone import utc
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('changeset', '0003_changeset_pdc_change_comment'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='changeset',
+            name='requested_on',
+            field=models.DateTimeField(default=datetime.datetime(2015, 9, 8, 2, 15, 31, 582918, tzinfo=utc)),
```

Comment:
Will this set existed records 'requested_on' to '2015-9-8'?
What will happen if theirs 'committed_on' are before that value?
I think 'requested_on' should allows to be Null and we could set 'duration' to NaN for historical data that without 'requested_on'.